e2e/selectors.ts

```diff
@@ -84,7 +84,7 @@ export const createSelectors = ({ screen, page, within }: Ctx) => {
         screen
           .getByTestId("panel-drawer")
           .within()
-          .findByText("Use abbreviations"),
+          .findByText("Use abbreviations", {}, { timeout: 10_000 }),
```

Comment:
🎉 for the test being fixed but do you know why it comes so slow ?

Reply:
Yes, this is because now the whole sidebar gets unmounted when segmentation component is changed, while previously this was not the case (see PROD) 👀  Maybe this could also be fixed by `keepPreviousData`, but for now it looks like we need to await this checkbox's presence 🤔

Reply:
OK thanks, I am going to take a look.

Reply:
Thank you! 🙇‍♂️